src/lib/workspace/SessionPanel.tsx

```diff
@@ -36,6 +37,18 @@ export function SessionPanel(props: Props) {
   const filteredEntities = uniq(
     sessionState.session?.filters.map((f) => f.entityId)
   );
+  const location = useLocation();
+  const [lastVarPath, setLastVarPath] = useState('');
+  const [lastVizPath, setLastVizPath] = useState('');
+  useEffect(() => {
+    console.log({ location });
```

Comment:
Were you planning to delete this `console.log` before merging, or keep it around for now?

Reply:
I was planning to delete, thanks!